test/extractExtension.unittest.js

```diff
@@ -0,0 +1,60 @@
+"use strict";
+
+const extractExtension = require("../lib/util/extractExtension");
```

Comment:
Can we avoid unit tests, they are useless, we need to tests real configurations, thanks

Reply:
hm, so should I just delete this test?
or should I replace it with test, that tests the real configurations?

Reply:
Better the second

Reply:
You can create small loader and pass these things to webpack, so you can test it easy

Reply:
ok, get it, I'll try to do it

Reply:
do you have any examples of such tests in the codebase? just trying to understand what I actually need to do

Reply:
Look at configuration tests

Reply:
get it, thanks

Reply:
@fromaline friendly ping, thank you